api-spec/api-spec.md

```diff
@@ -27,22 +27,24 @@ Items in the collection should match all filters to be returned when querying. T
 
 | Parameter      | Type          | Description        |
 | ------------ | ------------- | ---------------------- |
-| bbox | [number]       | Requested bounding box [west, south, east, north] |
-| time | string | Single date, date+time, or a range ('/' seperator), formatted to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6) |
+| ids | [string] | Array of Item ids to return. All other filter parameters that further restrict the number of search results (except `next` and `limit`) are ignored |
+| collections | [string] | Array of Collection IDs to include in the search for items. Only Items in one of the provided Collections will be searched |
 | intersects | GeoJSON Feature | Searches items by performing intersection between their geometry and provided GeoJSON Feature |
-| page | number | The page number of results. Defaults to 1 |
+| bbox | [number]       | Requested bounding box [west, south, east, north] |
+| time | string | Single date, date+time, or a range ('/' separator), formatted to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6) |
+| next | string | The token to retrieve the next set of results, e.g., offset, page, continuation token|
 | limit | number | The maximum number of results to return (page size). Defaults to 10 |
-| ids | [string] | Array of Item ids to return. All other filter parameters that further restrict the number of search results (except `page` and `limit`) are ignored |
-| collections | [string] | Array of Collection IDs to include in the search for items. Only Items in one of the provided Collections will be searched |
+
+In general, only one of **intersects** or **bbox** should be specified.  If both are specified, results should match both.
 
 ## STAC Endpoints
 
 STAC provides some additional endpoints for the root Catalog itself, as well as the capability to search the Catalog. Note that a STAC API does not need to implement WFS 3, in which case it would only support the endpoints given below. See the [OpenAPI specification document](definitions/STAC-standalone.yaml).
 
 | Endpoint      | Returns          | Description        |
-| ------------ | ------------- | ---------------------- |
-| /stac | Catalog        | Root catalog |
-| /stac/search | Items | GeoJSON FeatureCollection of Items found |
+| ------------- | ------------- | ---------------------- |
+| /stac         | Catalog        | Root catalog |
+| /stac/search  | ItemCollection   | GeoJSON FeatureCollection, probably containing additional STAC metadata, including search metadata from the `search` extension |
```

Comment:
Why is it called ItemCollection if it returns a GeoJSON FeatureCollection?

Reply:
See my comment above.  STAC defines "Item" and "ItemCollection" that are typed "Feature" and "FeatureCollection" to provide closer alignment with GeoJSON, however we are not technically 100% GeoJSON.

Reply:
I'd say we are a superset of GeoJSON. 100% compatible, with additions.

Reply:
I found a clear answer in the GeoJSON spec (hint: I actually read it) [https://tools.ietf.org/html/rfc7946#section-6] The term is "foreign members".  I think the correct description is "an Item is a GeoJSON Feature augmented with foreign members that contain the various values relevant to a STAC entity" -- likewise for ItemCollection/FeatureCollection.

Reply:
Looks like the only place we talk about ItemCollection is in the STAC openapi definition.  We should probably either add it into the existing Item spec, or add another spec on par with Item as we've done with Catalog and Collection -- I lean towards the latter. I think we need to be *very* clear when describing these to avoid confusing between Collection and ItemCollection.

Reply:
As said in #484: I'm not sure whether I would add this on the same level as Item / Collection / Catalog as those are all used in both static and dynamic catalogs, but ItemCollection is only used in API. Therefore, I'd be in favor of giving it a prominent place in the API spec - or is ItemCollection also getting an equivalent in static catalogs?

I like the statement on foreign members. Make sure to link "foreign members" to the GeoJSON spec!

Reply:
Added a description of ItemCollection and foreign members language to both Item and ItemCollection in my latest commit.